Directory.Build.props

```diff
@@ -17,11 +17,17 @@
         <InformationalVersion Condition=" '$(GitHubActionsRef)' == ''">$(SteamKitVersion) - Development</InformationalVersion>
         <InformationalVersion Condition=" '$(GitHubActionsRef)' != ''">$(SteamKitVersion) - CI (GitHub, ref: $(GitHubActionsRef))</InformationalVersion>
     </PropertyGroup>
+
     <PropertyGroup>
         <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
         <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
         <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
         <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
         <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
     </PropertyGroup>
-</Project>
\ No newline at end of file
+
+    <PropertyGroup>
+        <Deterministic>true</Deterministic>
+        <ContinuousIntegrationBuild Condition="'$(CI)' == 'true'">true</ContinuousIntegrationBuild>
```

Comment:
Isn’t this set automatically based on env cars these days?

Reply:
I don't think it is, I checked other packages that had Deterministic set, but nuget analyzer still complained about this not being set.